imgui-opengl/src/commonMain/kotlin/com/imgui/impl/ImGuiOpenGL3.kt

```diff
@@ -20,15 +20,7 @@ import io.ktor.utils.io.core.Closeable
 //  ES 3.0    300       "#version 300 es"   = WebGL 2.0
 //----------------------------------------
 
-expect class ImguiOpenGL3(
-		glslVersionStr: String = "#version 130",
-		useVertexArray: Boolean = true, // if !IMGUI_IMPL_OPENGL_ES2
-		unpackRowLength: Boolean = true,
-		usePolygonMode: Boolean = true,
-		useSamplerBinding: Boolean = false,
-		useClipOrigin: Boolean = true,
-		useDrawWithBaseVertex: Boolean = true
-) : Closeable {
+expect class ImGuiOpenGL3(glslVersionStr: String = "#version 130") : Closeable {
```

Comment:
This too.

Also, what happened to all the ctor variables?

Reply:
the constructor variables were all static version checks in imgui, so I reflected that.